<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opening_account</class>
 <widget class="QWidget" name="opening_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #34495e;</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>350</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام کاربری:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:25px;	</string>
   </property>
   <property name="text">
    <string>مشخصات را وارد کنید:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>400</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>مرز عبور:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>نام:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>250</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>کد ملی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>نام خوانوادگی:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>300</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	</string>
   </property>
   <property name="text">
    <string>تاریخ تولد:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birth_day">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_cod">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_name">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شماره حساب:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="number_account">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chose_number_account">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>انتخاب 
شماره حساب </string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_emploee_page">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>برگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>مبلغ افتتاح:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#ff0000;&quot;&gt; توجه: مبلغ اولیه نباید کمتر از 50/000 باشد&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="money_open">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>وضعیت حساب:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="creat_account">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

font-size:25px;
color:#EA2027; 
background-color:#bdc3c7; 
border: 2px solid #ff4d4d; </string>
   </property>
   <property name="text">
    <string>ایجاد حساب</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_war_balanc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_war_inf">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="accept">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ff5500;&quot;&gt;year/month/day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="accept_creat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cond">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>590</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:18px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="active">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:17px; color:yellow;</string>
   </property>
   <property name="text">
    <string>حساب فعال باشد؟</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
